cmd/admin-handlers.go

```diff
@@ -1509,16 +1505,26 @@ func (a adminAPIHandlers) TraceHandler(w http.ResponseWriter, r *http.Request) {
 		peer.Trace(traceCh, doneCh, trcAll, trcErr)
 	}
 
-	enc := json.NewEncoder(w)
+	keepAliveTicker := time.NewTicker(500 * time.Millisecond)
+	defer keepAliveTicker.Stop()
+
 	for {
 		select {
 		case entry := <-traceCh:
-			if err := enc.Encode(entry); err != nil {
+			entryBytes, err := json.Marshal(entry)
```

Comment:
Is there any reason to move away from using `json.Encoder`?

Reply:
It was a mistake @krisis changed